tests/test_data/theglutenfreeaustrian.com/theglutenfreeaustrian_1.json

```diff
@@ -0,0 +1,44 @@
+  "instructions_list": [
+    "How to make a Gluten Free Sourdough StarterStart: to a clean, sanitized mason jar add 25 grams of gluten free flour and 25 grams warm water (around 77F/25C – 84F/29C). Using a plastic spoon or spatula mix it together. Cover it with cheesecloth and a rubber pan or loosly cover it with the mason jar lid. Allow for it to sit at room temperature at the counter.",
+    "Feeding #1: Around 8-12hrs after starting the sourdough starter, add 25 grams of gluten free flour and 25 grams of warm water. Mix it together, loosely cover it and allow for it to see at room temperature . If your mixture seems very stiff, add 5-10 extra grams of water to loosen up the mixture. It’s okay for the starter to be too runny vs being too dry.",
+    "Feeding #2: 8-12 hours after the first feeding, it’s time for the second meal. Depending on which flour you used, your may already see some bubbles and growth in your starter. Add 25 grams of flour and 25 grams of warm water, mix, cover and place in a warm place.",
+    "Feeding #3: You should definitely see some small bubbles and growth in your gluten free sourdough starter now. Repeat the feeding, mix, cover and store at ambient temperature.",
+    "Feeding #4: On day three it's time to switch to a 1:1:1 feeding process and start discarding some of the starter. Remove all but 50 grams of the sourdough starter from the jar. Compost the discard or throw it away. Add 50 grams of fresh flour and 50 grams of warm water (around 77F/25C - 84F/29C) to the jar and mix. Cover and store at ambient temperature. (At this point, it is too early to save the discard for recipes since it is not yet from an established starter. )",
+    "Feeding #5 up to 7 days: The sourdough starter should become more active now. You may also see bubbles. Keep up with the feedings every 8-12hrs. Remove all but 50 grams of the sourdough starter from the jar. Compost the discard or throw it away. Add 50 grams of fresh flour and 50 grams of warm water (around 77F/25C - 84F/29C) to the jar and mix. Cover and store at ambient temperature. ",
+    "After Seven days: After seven days and at least three days of consistent growth and doubling the size after every feeding within 4-12 hours, the gluten-free sourdough starter is ready to be used in recipes. I recommend feeding the sourdough starter ONCE a day for an additional week. This will strengthen the starter and build a strong starter to have on hand. I call those feedings maintenance feedings, which can be done with less flour and less often."
+  ],
+  "category": "Gluten Free Bread",
+  "yields": "250 servings",
+  "description": "This simple gluten free sourdough starter is made from two ingredients: a gluten free flour, such as sorghum flour, and filtered water. Small batch gluten free sour dough starter recipe",
+  "total_time": 10090,
```

Comment:
Just noting a small quirk here -- but it is correct: this `10090` value is from `P7DT10M` (seven days plus ten minutes).

Reply:
Not many recipes have total_time spanning multiple days so it good to see working as expected!